lib/rubocop/server/core.rb

```diff
@@ -56,7 +56,8 @@ def server_mode?
       def start_server(host, port)
         @server = TCPServer.open(host, port)
 
-        puts "RuboCop server starting on #{@server.addr[3]}:#{@server.addr[1]}."
+        (ARGV.include?('--stderr') ? $stderr : $stdout)
+          .puts "RuboCop server starting on #{@server.addr[3]}:#{@server.addr[1]}."
```

Comment:
Can you refactor it?

```suggestion
        output_stream = ARGV.include?('--stderr') ? $stderr : $stdout

        output_stream.puts "RuboCop server starting on #{@server.addr[3]}:#{@server.addr[1]}."
```